examples/demo.css

```diff
@@ -69,12 +69,14 @@ body > .container-fluid {
   min-width:300px;
 }
 
-.hover {
-  background: #eee;
+.import-image-zone {
+  margin: 10px auto 30px auto;
+  max-width: 230px;
+  min-width: 230px;
 }
 
-.dropzone input {
-  max-width: 100%;
```

Comment:
Can we just keep the max-width: 100 %. If yes, then also change max-width  of image-import-zone accordingly